### ISSUE TITLE
Unexpected unknown property (property-no-unknown)

### ISSUE THREAD

**Issue by shripadk** [styled-components/stylelint-processor-styled-components#13]
Not sure if this is related to #6 (please close if so) but it breaks with the following complex interpolation:

```javascript
  const Button = styled.button`
    margin-${props => ((props.theme.dir === 'rtl') ? 'left' : 'right') }: 12.5px;
  `;
```

This is used for providing RTL support where the **dir** property can be set in `ThemeProvider` like so:

```javascript
<ThemeProvider theme={{ dir: 'rtl' }}>
  <Button>Hello, World!</Button>
</ThemeProvider>
```

Stylelint throws the following error: _Unexpected unknown property "margin-$undefined" (property-no-unknown)_.

**Comment by mxstbr** [comment]
Ah interesting, that's a bug that'll be interesting to fix! Mind submitting a PR with a failing test case? That'd be super helpful.

**Comment by shripadk** [comment]
Have submitted a PR with failing test cases. Let me know if you need anything else :)

**Comment by emilgoldsmith** [comment]
This should be able to be handled with an `sc-custom` tag as added in #70 when we publish it in the near future.
